exe/Main.hs

```diff
@@ -11,7 +11,7 @@ import GHC.Generics (Generic)
 
 main :: IO ()
 main = do
-  let stmt = "select count(*), sysdate, 'ignore next column', 125.24, 3.14 from dual"
+  let stmt = "select count(*), sysdate, 'ignore next column', 125.24, 3.14, null from dual"
```

Comment:
I couldn't get this example to work unless I explicitly cast the null as a `NUMBER (38,0)`